debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/transforms/ExtractNewDocumentState.java

```diff
@@ -5,6 +5,19 @@
  */
 package io.debezium.connector.mongodb.transforms;
 
+import io.debezium.config.Configuration;
```

Comment:
Seems like you applied some auto-organization of imports? Can you please revert this, to keep the diff focused on the actual changes? You can run `mvn process-sources` to ensure the formatting is right before committing. Thx!

Reply:
yes.. My intellij reformat is doing that.. Fixed it now. Thanks

Reply:
No worries; looking much better now, thx!